dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpChannelConfig.java

```diff
@@ -866,6 +871,87 @@ private void parseH2MaxFrameSize(Map<Object, Object> props) {
         }
     }
 
+    private void parseH2ConnCloseTimeout(Map<?, ?> props) {
+        Object value = props.get(HttpConfigConstants.PROPNAME_H2_CONN_CLOSE_TIMEOUT);
+        if (null != value) {
+            try {
+                this.http2ConnectionCloseTimeout = convertLong(value);
+                if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
+                    Tr.event(tc, "Config: H2 Connection Close timeout is " + getH2ConnCloseTimeout());
+                }
+            } catch (NumberFormatException nfe) {
+                FFDCFilter.processException(nfe, getClass().getName() + ".parseH2ConnCloseTimeout", "1");
+                if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
+                    Tr.event(tc, "Config: Invalid H2 Connection Close Timeout of " + value);
+                }
+            }
+        }
+    }
+
+    // Stream level default initial window size for all new streams, goes out in preface settings frame
+    private void parseH2SettingsInitialWindowSize(Map<?, ?> props) {
+        Object value = props.get(HttpConfigConstants.PROPNAME_H2_SETTINGS_INITIAL_WINDOW_SIZE);
+        if (null != value) {
+            try {
+                if (convertLong(value) > Integer.MAX_VALUE) {
```

Comment:
The metatype defines this as type Integer with min and max values, do we need to have this check here?

Reply:
I've removed the method with the checks.  Without it, the config error is still reported.